recipes/r-ggrepel/meta.yaml

```diff
@@ -2,14 +2,14 @@ package:
   name: r-ggrepel
   # Note that conda versions cannot contain -, so any -'s in the version have
   # been replaced with _'s.
-  version: "0.5"
+  version: "0.6.5"
 
 source:
-  fn: ggrepel_0.5.tar.gz
+  fn: ggrepel_0.6.5.tar.gz
   url:
-    - http://cran.r-project.org/src/contrib/ggrepel_0.5.tar.gz
-    - http://cran.r-project.org/src/contrib/Archive/ggrepel/ggrepel_0.5.tar.gz
-
+    - http://cran.r-project.org/src/contrib/ggrepel_0.6.5.tar.gz
```

Comment:
In general this looks great. Feel free to clean up the comments and please add a checksum for this package.